### PR TITLE
Mod

### PR DESCRIPTION
**Pull request by s-rosenzweig**
Added two extras:
1. Loop button which allows to loop a dimension (e.g. useful for cardiac visualisation)
2. Specify dimensions to show at startup

Moreover:
Consistency to the dimension in file-naming for save-dialog

### CODE REVIEW

File: src/iopts.c [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/* Copyright 2015-2017. The Regents of the University of California.
```
Comment (uecker) [inline src/iopts.c:1]: This should go into BART - not the viewer. Also please do not use public github pull requests but our internal git.
- The copyright header make no sense.

File: src/view.c [hunk 2]
```diff
@@ -601,17 +605,24 @@ extern gboolean draw_callback(GtkWidget *widget, cairo_t *cr, gpointer data)
...
 {
 	long sq_dims[2] = { 0 };
 
 	int l = 0;
 
-	for (int i = 0; (i < DIMS) && (l < 2); i++)
-		if (1 != dims[i])
-			sq_dims[l++] = i;
+	// Check if dimensions to display are defined
+	if (showDims[0] != -1 && showDims[1] != -1) {
```
Comment (uecker) [inline src/view.c:620]: please follow bart coding style. brackets around the comparisons,

File: src/view.c [hunk 3]
```diff
@@ -627,7 +638,12 @@ struct view_s* create_view(const char* name, const long pos[DIMS], const long di
 	v->pos = xmalloc(DIMS * sizeof(long));
 
 	for (int i = 0; i < DIMS; i++)
-		v->pos[i] = (NULL != pos) ? pos[i] : 0;
+			v->pos[i] = (NULL != pos) ? pos[i] : 0;
```
Comment (uecker) [inline src/view.c:647]: there is a tab too much

File: src/view.c [hunk 4]
```diff
@@ -627,7 +638,12 @@ struct view_s* create_view(const char* name, const long pos[DIMS], const long di
...
+
+	if ( iopts.r > 0) {
```
Comment (uecker) [inline src/view.c:649]: here is a space too much and in the next lines there are spaces missing around the operators

File: src/draw.c [hunk 5]
```diff
@@ -303,17 +303,17 @@ extern char* get_spec(int i)
...
-		case  5: return "n"; break;
-		case  6: return "o"; break;
-		case  7: return "p"; break;
-		case  8: return "q"; break;
-		case  9: return "slice"; break;
+		case  5: return "te"; break;
+		case  6: return "coeff"; break;
+		case  7: return "coeff2"; break;
+		case  8: return "iter"; break;
+		case  9: return "cshift"; break;
```
Comment (uecker) [inline src/draw.c:310]: I am not so much a fan of this naming concention. Also some stuff is outdated (cshift).

**Comment by s-rosenzweig** [comment]
Will use internal git

**Pull request closed without merging**